app/src/full/java/io/homeassistant/companion/android/notifications/MessagingService.kt

```diff
@@ -433,6 +446,12 @@ class MessagingService : FirebaseMessagingService() {
                 if (title == TURN_ON)
                     bluetoothAdapter.enable()
             }
+            COMMAND_BLE_TRANSMITTER -> {
+                if (title == TURN_OFF)
+                    BluetoothSensorManager.enableDisableBLETransmitter(applicationContext, false)
+                if (title == TURN_ON)
+                    BluetoothSensorManager.enableDisableBLETransmitter(applicationContext, true)
+            }
```

Comment:
Hi @dshokouhi Apologies I couldn't resist the urge to add something else :) adding a Command Notification to allow people to turn of Ble Transmitter as required from HA.  I think this might away with future requirements I was considering to add a listener to re-enable transmitter on bluetooth (re) start, as we can orchestrate that from HA. It also adds other possibilities as well, like turning off Transmitter to save battery when not required.

The code looks pretty straight forward and I've added a separate commit to MessagingService.kt, in case we want to remove it from this PR. Also a separate PR to docs project under home-assistant/companion.home-assistant#451 .

The only snag that I've got is that I haven't been able to test it - I've been looking at  getting firebase set up with https://github.com/home-assistant/mobile-apps-fcm-push but haven't succeeded so far (running into script running permissions issue that I can't solve as don't have full access).  I was wondering if we'd consider adding to a beta as is, or if you could suggest any  other way of working round the problem. 

Thanks,